### PR TITLE
updated logging when asset builders fail to start to have a bit more details

### PR DESCRIPTION
**Pull request by AMZN-stankowi**

### CODE REVIEW

**Review by amzn-mike — APPROVED** [review]

**Comment by srikappa-amzn** [comment]
@AMZN-stankowi , can you please provide next actions on this PR or close it accordingly?

**Comment by lmbr-pip** [comment]
@AMZN-stankowi Can we update the description with some details including how tested?

**Comment by AMZN-stankowi** [comment]
> @AMZN-stankowi , can you please provide next actions on this PR or close it accordingly?

I've been out of town, I plan to refresh this when I have time.

**Comment by monroegm** [comment]
@AMZN-stankowi Do you still want to keep this open for now?

File: Code/Tools/AssetProcessor/native/utilities/BuilderManager.cpp [hunk 1]
```diff
@@ -86,10 +86,10 @@ namespace AssetProcessor
                 AZ_Error("Builder", false, "AssetBuilder failed to connect within %d seconds", s_StartupConnectionWaitTimeS);
             }
 
-            return false;
+            return AZ::Failure(AZStd::string("Connection failed, see previous error messages for details."));
```
[the remark this answers is not in the document]
Comment (AMZN-stankowi) [reply Code/Tools/AssetProcessor/native/utilities/BuilderManager.cpp:89]: Done, fixed

File: Code/Tools/AssetProcessor/native/utilities/BuilderManager.cpp [hunk 2]
```diff
@@ -162,7 +162,7 @@ namespace AssetProcessor
 
         if (!m_processWatcher)
         {
-            return false;
+            return AZ::Failure(AZStd::string("Process watcher failed to start. See previous log messages for details."));
```
[the remark this answers is not in the document]
Comment (AMZN-stankowi) [reply Code/Tools/AssetProcessor/native/utilities/BuilderManager.cpp]: Done, fixed

**Review by lmbr-pip — APPROVED** [review]
Please update the description for PR

**Pull request merged**